Mathlib/CategoryTheory/Localization/DerivabilityStructure/Basic.lean

```diff
@@ -126,6 +126,59 @@ instance [W₁.ContainsIdentities] : (LocalizerMorphism.id W₁).IsRightDerivabi
   dsimp
   exact TwoSquare.guitartExact_id W₁.Q
 
+/-- A localizer morphism `Φ : LocalizerMorphism W₁ W₂` is a left derivability
+structure if it has left resolutions and the 2-square where the top and bottom functors
+are localizations functors for `W₁` and `W₂` are Guitart exact. -/
+class IsLeftDerivabilityStructure : Prop where
+  hasLeftResolutions : Φ.HasLeftResolutions := by infer_instance
+  guitartExact' : TwoSquare.GuitartExact ((Φ.catCommSq W₁.Q W₂.Q).iso).inv
```

Comment:
Is there a reason why this is primed?

Reply:
Thanks for the reviews! Yes, because we do no want to apply this property directly: the more useful lemma is `guitartExact_of_isLeftDerivabilityStructure` which apply not only to the "constructed" localization functors `W₁.Q` and `W₂.Q` but to any tuple of localization functors.

Reply:
I'm not sure why the docPrime linter is not firing here but can you add a comment saying "please use `guitartExact_of_isLeftDerivabilityStructure` instead"? Thanks.